solana/pyth2wormhole/program/src/attest.rs

```diff
@@ -248,15 +262,30 @@ pub fn attest(ctx: &ExecutionContext, accs: &mut Attest, data: AttestData) -> So
     );
     solana_program::program::invoke(&transfer_ix, ctx.accounts)?;
 
+    let payload = batch_attestation.serialize().map_err(|e| {
+        trace!(&e.to_string());
+        ProgramError::InvalidAccountData
+    })?;
+
+    // Adjust message account size if necessary
+    if !accs.wh_message.is_initialized() && accs.wh_message.payload.len() != payload.len() {
+        // Learn how much we're adjusting (Note: assuming surrounding
+        // Wormhole message type is constant-size).
+        // positive -> grow,
+        // negative -> shrink,
+        let diff = payload.len() as i64 - accs.wh_message.payload.len() as i64;
+        accs.wh_message.0 .0 .0.realloc(
+            (accs.wh_message.payload.len() as i64 + diff) as usize,
+            false,
+        )?;
```

Comment:
Am I missing something (likely!) or is this equivalent to just doing `accs.wh_message.0 .0 .0.realloc(payload.len(), false)`?